unearthed-arcana/2017/20170206.xml

```diff
@@ -300,64 +300,40 @@
       <element name="Stone’s Durability" type="Archetype Feature" source="Unearthed Arcana: Sorcerer" id="ID_WOTC_UA_ARCHETYPE_FEATURE_STONE_SORCERER_STONES_DURABILITY">
         <description>
           <p>At 1st level, your connection to stone gives you extra fortitude. Your hit point maximum increases by 1, and it increases by 1 again whenever you gain a level in this class.</p>
-          <p class="indent">As an action, you can gain a base AC of 13 + your Constitution modifier if you aren’t wearing armor, and your skin assumes a stony appearance. This effect lasts until you end it as a bonus action, you are incapacitated, or you don armor other than a shield.</p>
+          <p class="indent">As an action, you can gain a base AC of {{ac:stones durability}} if you aren’t wearing armor, and your skin assumes a stony appearance. This effect lasts until you end it as a bonus action, you are incapacitated, or you don armor other than a shield.</p>
         </description>
```

Comment:
The description is used for display in the application, don't use any replacements here. Only in the sheet descriptions.